contents/blog/what-to-ask-in-interviews.md

```diff
@@ -0,0 +1,114 @@
+
+This just doesn't feel like it should be normal. People should be able to reduce the chances they end up in a situation like [this](https://twitter.com/carnage4life/status/1532472869334110208), or [this](https://nypost.com/2022/05/26/twitter-rescinds-job-offer-to-new-hire/).
+
+> Disclaimer: These questions are quite direct, but a company that reacts badly to them may not be a good place to work. There are also a _lot_ of questions here - think of them as themes, and you don't need to ask them all. Prioritize based on what you hear through the process.
+
+If I were job hunting, these are the themes I'd be interested in - it's skewed towards (i) startup / early growth software companies, (ii) engineers and (iii) interviews with founders (this is what I know!), but the concepts feel like they'd be useful beyond that.
+
+## Does the company have product-market fit?
+
+This is the single most important thing a company can do to survive and grow.
+
+"Do you ever question if you have product-market fit?"
+"When did you reach product-market fit? How did you know?"
+"What do you need to do to get to product-market fit?"
+"What's your revenue? What was it a year ago?"
+"How many daily active users do you have?"
+
+It's ok if these answers show you the founder doesn't have product-market fit. In this case, figure out if they will get to a yes. Unless you want to join a sinking ship, of course!
+
+Early stage founders are (or should be) super-mega-extra-desperately keen to have product market fit - it's all that really matters. The ones that will succeed are those that are honest about this (or those that have it already) and are prioritizing it. Many will think or say (intentionally or through self-delusion) that they have it when they don't. Low user or revenue numbers and vague answers to the example questions above are a sign that it isn't there. Product market fit [is _very_ obvious](https://www.ycombinator.com/library/5z-the-real-product-market-fit).
+
+Avoid:
+
+* Founders who aren't spending a ton of time talking and selling to users pre-product-market fit - it's a flag to attempt to outsource/delegate this
+* Companies where you don't understand the product or its benefits - if it can't be explained to you, others will struggle
+* Solutions looking for a problem
+
+If there is no product-market fit, but you think the team could find it, it's very important the company has lots of runway (it'll be hard to raise more money without product-market fit, so time to find it is crucial), that the startup doesn't have a huge team (it's harder to make rapid changes in direction with a lot of people), and that the company is fast (look out for bureaucracy, management, or a team that can't work well with ambiguity).
+
+## How much runway does the company have? Does their spending look within reason?
+
+"Are you [default alive](http://www.paulgraham.com/aord.html)?"
```

Comment:
'Default alive' - is this a bit too niche/YC as terminology? I had never heard of it until a couple of weeks ago. 

An alternate way to frame this could be 'If you stopped spending more money today, would the company survive based on current revenue growth?'

Reply:
I prefer the default alive here, tbh. Any alternatives are just a bit long winded.

Reply:
Fair comment but I think the link gives more explanation (and I'm writing this with an HN crowd in mind)